gulpfile.js

```diff
@@ -125,6 +129,27 @@ gulp.task('generate-index-files', function() {
     .pipe(gulp.dest('./dist'));
 });
 
+gulp.task('beautify', function () {
+  gulp.src('dist/**/*.js')
+      .pipe(beautify({ config: 'node_modules/mofo-style/linters/.jsbeautifyrc' }))
+      .pipe(gulp.dest('dist'));
+});
+
+gulp.task('jshint', function() {
+  return gulp.src('./lib/*.js')
+      .pipe(jshint({ lookup: 'node_modules/mofo-style/linters/.jshintrc' }))
+      .pipe(jshint.reporter('default'));
+});
+
+
+gulp.task('jscs', function () {
+  // jscs doesn't play nice with *.jsx files so we're avoiding lib/*.jsx
```

Comment:
Ah that's a bummer--we should file that as an issue once this lands, so we don't forget!